users/discord/utils/util.js

```diff
@@ -19,3 +19,511 @@ export const getUsers = async (tab) => {
     console.error(err);
   }
 };
+// mock user data
+export const mockUsersData = {
+  in_discord: [
+    {
+      id: 'user_1',
+      discordId: '123456789',
+      username: 'Alice',
+      avatar: 'https://placekitten.com/50/50?image=1',
+    },
```

Comment:
Hey so for development, using mock data was suggested. 
But not to push mock data code, in the PR
Make API calls according to the contract decided
and put it behind a feature flag

Reply:
I have removed the mock data and added the feature behind feature flag.